source/initial_conditions/solidus.cc

```diff
@@ -24,7 +24,7 @@
 #include <aspect/geometry_model/spherical_shell.h>
 #include <aspect/boundary_temperature/interface.h>
 
-#include <boost/math/special_functions/spherical_harmonic.hpp>
+#include <boost/lexical_cast.hpp>
```

Comment:
`lexical_cast` is used in the parsing of these plugins. It seems that when I removed the `#include` of `boost::spherical_harmonic` that function was no longer available, so I re-included it.